vendor/github.com/rs/rest-layer-hystrix/hystrix.go

```diff
@@ -0,0 +1,148 @@
+// Package restrix is a REST Layer resource storage wrapper to add hystrix support
```

Comment:
Yeah I saw. What you suggest makes sense, but I don't mind if you want to keep it as is for now.